src/EnergyPlus/WaterThermalTanks.cc

```diff
@@ -3612,7 +3612,7 @@ namespace WaterThermalTanks {
 								ShowContinueError( "Please leave the source side inlet and outlet fields blank." );
 								ErrorsFound = true;
 							} else {
-								WaterHeaterSaveNodes &HPWHNodeNames = HPWHSaveNodeNames( NumHeatPumpWaterHeater );
+								WaterHeaterSaveNodes &HPWHNodeNames = HPWHSaveNodeNames( HPWaterHeaterNum );
```

Comment:
Was looking up the water heater node names using the variable for "total number of heat pump water heaters" when it should have been looking it up by the variable for "current heat pump water heater number". Worked fine when there was only one HPWH in the idf (`NumHeatPumpWaterHeater == HPWaterHeaterNum`).